.eslintrc.json

```diff
@@ -0,0 +1,23 @@
+{
```

Comment:
Once you rebase this PR, be sure to lint the /scripts folder, since I expect many formatting improvements will happen in there. It was hard to work without ESLint, so I'm glad you've added it.